### PR TITLE
[WIP]: Making PyKE pyraf-free and python3 compatible

### PR DESCRIPTION
**Pull request by mirca**

### CODE REVIEW

**Comment by mirca** [comment]
@barentsen marking you here XD

File: pyke/kepstat.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+def mean_err(array):
+    return math.sqrt(np.sum(array * array)) / len(array)
+
+
+def rms(array1, array2, logfile, verbose):
+    """root mean square of two arrays"""
+    if len(array1) != len(array2):
+        message  = ("ERROR -- KEPSTAT.RMS: Arrays have unequal sizes - "
+                    "array1 = {0}, array2 = {1}".format(len(array1),
+                                                        len(arra2)))
```
Comment (mirca) [inline pyke/kepstat.py]: ``arra2`` --> ``array2``

**Comment by gully** [comment]
Update the `readme.md` of py3-dev to remove reference to astro-conda and pyraf, and indicate the preferred `pip install -e .` command line arguments.

**Comment by mirca** [comment]
@barentsen @gully take a look at this whenever you have time and let me know your feelings ;) I plan to do a release in the next few days according to your reviews.

Docs can be generated and visualized as follows:
```
python setup.py build_docs

cd docs
make html
firefox build/html/index.html
```

Thanks!!!

**Comment by mirca** [comment]
@barentsen @gully I'm gonna merge this as is since a lot of changes are related to moving/deleting/reorganizing files.

All tools are now compatible with python 3 and the documentation has been moved to Sphinx =)

PS: This is gonna be our ``master`` version. I've moved the ``pyraf`` version to the branch ``py27-pyraf``.

**Comment by gully** [comment]
This all sounds good, I'm delinquent on trying out many of the tools, but I will experiment in the week ahead.

**Comment by barentsen** [comment]
:+1: Thanks for the merge @mirca!  I agree this looks stable enough to be the new master.  I have been using the master branch today and will start opening issues for possible improvements.

**Comment by mirca** [comment]
@gully @barentsen cool, thank you guys for reviewing this!

**Pull request merged**